programs/dynamic-bonding-curve/src/instructions/partner/ix_create_config.rs

```diff
@@ -46,8 +94,11 @@ pub struct ConfigParameters {
     pub token_update_authority: u8,
     pub migration_fee: MigrationFee,
     pub migrated_pool_fee: MigratedPoolFee,
+    pub partner_impermanent_locked_lp_info: LpImpermanentLockInfoParams,
+    pub creator_impermanent_locked_lp_info: LpImpermanentLockInfoParams,
```

Comment:
that is fine to move to new endpoint create_config2 if we requires more space, and refactor a bit for params, because some of params are only available for damm v2 
Or maybe `create_config2` will only create config to migrate to damm v2, so it will be easier to use